Content.Client/DistortionMap/DistortionMap.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Collections.Generic;
+using Robust.Client.Graphics;
+using Robust.Shared.Enums;
+using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Maths;
+
+namespace Content.Client.DistortionMap;
+
+class DistortionMapOverlay : Overlay
+{
+    [Dependency] private readonly IClyde _clyde = default!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+
+    public override OverlaySpace Space => OverlaySpace.WorldSpace;
```

Comment:
If [the coordinate stuff gets worked out](https://github.com/space-wizards/space-station-14/pull/5698/files#r762446848), this could live in any of the later spaces as well.